src/components/ModelTableList/_model-table-list.scss

```diff
@@ -65,7 +65,7 @@
         width: 100%;
       }
 
-      .model-access-alt {
+      .has-permission .model-access-alt {
```

Comment:
if the user `has-permission` then don't show it? :)

Reply:
Don't show the model access button alternative? Yes, because we don't want to show the button _and_ the alternative if user has permission.